proto/process/agent.proto

```diff
@@ -1121,6 +1127,11 @@ message ResourceRequirements{
 	ResourceRequirementsType type = 4;
 }
 
+// ResourceMetrics is used in backend enrichment.
+message ResourceMetrics {
+	map<string, double> metricValues = 1;
```

Comment:
its only used in the backend, but wdyt about adding an example comment? An example for an explicit metric string since `string` is broad?

Reply:
Is it useful to mention how the metric in the backend look like in that public repo? That's merely a simple map holding a metric key and a metric value, I was thinking it's good enough without further details (also the BE can change over time).

Reply:
That's really just a map holding a metric key and a metric value so I was thinking it's self-explanatory. Our internal documentation should be the place where further details about the usage is available (especially since details of the BE implementation could change over time).

Reply:
I am bit torn here. I do understand your point, but since it's a shared proto we can use the code comment instead, saving us a lookup.

In any case it s a nit

Reply:
Alright. I'll lean on the cautious side for now since I'm not sure about revealing backend implementation details in a public repo, and by the looks of it, other backend-only fields have vague comments.

In any case I don't expect this to have much impact since no consumer is expected for that part of the message.